cypress/e2e/create-simple-config.cy.ts

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2020 - 2025 Pionix GmbH and Contributors to EVerest
+
+/// <reference types="cypress" />
+
+describe("create a simple config", () => {
+  it("should create a config successfully", () => {
+    cy.visit("/");
+    cy.contains("div", "Simulator / Mock").click();
+    cy.get(".mdi-plus").click();
+    cy.get("input").type("simple-config");
+    cy.get(".mdi-check").click();
+    cy.contains("div", "EvseManager").click();
+    cy.contains("div", "YetiDriver").click();
```

Comment:
If possible, use more precise selectors. Right now these will break the test as soon as something about the html layout changes. I know it is hard whenever it has something to do with konvajs, so some of these hacks might be necessary, but e.g. the input one: This works as long as there is only one input element. Instead you can use the convention to add [data-cy] selectors, see here: https://docs.cypress.io/app/core-concepts/best-practices (search for data-cy)